src/com/xilinx/rapidwright/timing/TimingModel.java

```diff
@@ -500,17 +502,82 @@ protected List<TimingGroup> determineGroups(List<Node> nodes, List<IntentCode> n
         return result;
     }
 
+    private Pair<Integer,Integer> findReferenceTileLocation(Device d) {

```

Comment:
The instance of this class has a reference to a `Device` object.  Should this method be static or can we remove parameter?

Reply:
Will remove the parameter and use from the class member.